### ISSUE TITLE
Lightning Network

### ISSUE THREAD

**Issue by the9ull** [opencap/protocol#56]
Cryptocurrency payments are moving from on chain transactions to second layer.
A protocol to simplify payments must take care of Lightning Network. Lightning Network is evolving, so could be too early to define a complete interface.
I think that it is possible to provide basic information of a node:
* pubkey
* host:port

Lightning should provide in the next future the possibility to send money to a node, so this would be useful to gather donations.

Alternately, providing pubkey@host:port of a ln node could be enough, because any further communication could directly through the ln node.

**Comment by lane-c-wagner** [comment]
Agreed. Are you familiar with lightning invoices? Is there a specific URI or something we can use? Might be easier to chat about this is discord btw

**Comment by the9ull** [comment]
[The BOLT 11](https://github.com/lightningnetwork/lightning-rfc/blob/master/11-payment-encoding.md) describes payment requests.
I'm not following the developement.
This is an invoice generated with lnd.

```json
{
    "destination": "030c3f19d742ca294a55c00376b3b355c3c90d61c6b6b39554dbc7ac19b141c14f",
    "payment_hash": "619e84d4e29c09b7406561e53ed1bf475335a284b2af47e567ed6a33c661a9ef",
    "num_satoshis": "1000",
    "timestamp": "1542706715",
    "expiry": "3600",
    "description": "",
    "description_hash": "",
    "fallback_addr": "",
    "cltv_expiry": "144",
    "route_hints": [
    ]
}
```
